plugins/repository-s3/qa/amazon-s3/build.gradle

```diff
@@ -57,26 +78,34 @@ task s3Fixture(type: AntFixture) {
     dependsOn compileTestJava
     env 'CLASSPATH', "${ -> project.sourceSets.test.runtimeClasspath.asPath }"
     executable = new File(project.runtimeJavaHome, 'bin/java')
-    args 'org.elasticsearch.repositories.s3.AmazonS3Fixture', baseDir, s3Bucket
+    args 'org.elasticsearch.repositories.s3.AmazonS3Fixture', baseDir
```

Comment:
I'd prefer if we continue to pass the bucket names to the fixture so that we can reuse it in different projects more easily.

Reply:
Ok, I've reverted that. There is, however, already some other coupling (duplication of string values) between the `build.gradle` file and the fake S3 fixture that would need attention to use this fixture elsewhere.